--- config.def.h	2024-01-17 15:05:42.787060226 -0700
+++ config.def.patch.h	2024-01-17 15:05:42.790393560 -0700
@@ -64,6 +64,17 @@
  * wifi_perc           WiFi signal in percent          interface name (wlan0)
  */
 static const struct arg args[] = {
-	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	/* function          format         argument */
+        { run_command,       " %s%%v | ",   "special-keys volume" },
+        { run_command,       "%s | ",       "special-keys volume status" },
+        { run_command,       "%s%%l | ",    "special-keys backlight" },
+        { wifi_perc,         "%s%%w | ",    "wlp108s0" },
+        { wifi_essid,        "%s | ",       "wlp108s0" },
+        { battery_perc,      "%s%%b | ",    "BAT1" },
+        { battery_remaining, "%s",          "BAT1" },
+        { battery_state,     "%s | ",       "BAT1" },
+        { cpu_perc,          "%s%%c | ",    NULL },
+        { ram_perc,          "%s%%m | ",    NULL },
+        { disk_perc,         "%s%%d | ",    "/" },
+        { datetime,          "%s ",         "%F %T" },
 };
